frontend/src/scenes/session-recordings/player/playlist-popover/playlistPopoverLogic.ts

```diff
@@ -108,11 +108,12 @@ export const playlistPopoverLogic = kea<playlistPopoverLogicType>([
                 actions.reportRecordingPlaylistCreated('pin')
 
                 if (!newPlaylist) {
-                    // This indicates the billing popover has been shown so we should close the modal
+                    // This indicates the billing popover has been shown, so we should close the modal
                     actions.setShowPlaylistPopover(false)
                     return
                 }
 
+                // TODO change to currentPlaylists isn't recalculating the selector
```

Comment:
should we address this on this PR or raise a new issue? so it's not forgotten in comments (considering its a bug?).